clients/src/main/resources/common/message/FindCoordinatorRequest.json

```diff
@@ -23,12 +23,16 @@
   // Version 2 is the same as version 1.
   //
   // Version 3 is the first flexible version.
-  "validVersions": "0-3",
+  //
+  // Version 4 adds CoordinatorKeys
+  "validVersions": "0-4",
   "flexibleVersions": "3+",
   "fields": [
-    { "name": "Key", "type": "string", "versions": "0+",
+    { "name": "Key", "type": "string", "versions": "0-3",
```

Comment:
Just for my understanding, do we have to keep the old field here to be able to serve requests from older versioned clients?

Reply:
Yes we have to keep the existing fields to handle all versions